src/daemon/globals.c

```diff
@@ -1,13 +1,38 @@
+/**
+ * collectd - src/globals.c
+ * Copyright (C) 2017  Florian octo Forster
```

Comment:
I think the OSPO would want you to put "Google, Inc." here. Dito for `globals.h`.

Reply:
Gotcha. I put "Google LLC" because of the recent name change. Is that right?